### DISCUSSION TITLE
Too Many Requests

### DISCUSSION THREAD

**Discussion by mansan1230** [Rapptz/discord.py#7755] [Q&A] [answered]
discord.errors.HTTPException: 429 Too Many Requests (error code: 0): You are being blocked from accessing our API temporarily due to exceeding our rate limits frequently. Please read our docs at https://discord.com/developers/docs/topics/rate-limits to prevent this moving forward.

How can I solve this problem ??
i am running my bot in relp,it

**Comment by joannx** [comment, marked as answer]
replit uses a shared ip, meaning if one person goes down, everyone goes down. you have two options to solve this

wait until the temporary block is over
or
dont use replit, theres many vpses that are super cheap

**Reply by mansan1230** [reply]
so the problem is not caused by my code ???
it is by other people??

**Reply by jelni** [reply]
Yes, don't use replit to host Discord bots. If you want free hosting, I personally recommend Oracle Cloud which has a great lifetime free tier.

**Reply by mansan1230** [reply]
okok thx ,   i will try it
